### PR TITLE
Use shell for loop then ansible with_items in tf benchmarks cases run…

### PR DESCRIPTION
**Pull request by liu-sheng**
For some unkonwn reason, Zuul cannot dynamically flush the log stream
in a ansible task running with "with_items", so change it to use shell
for loop.

### CODE REVIEW

File: playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml [hunk 1]
```diff
@@ -133,76 +133,76 @@
       shell: |
         set -o pipefail
         set -ex
-        kubectl -n kf-tfbenchmarks create -f "{{ ansible_user_dir }}/workspace/kf-tfbenchmarks/{{ item }}"
-        sleep 20
-        kubectl -n kf-tfbenchmarks get tfjobs -o wide
-        kubectl -n kf-tfbenchmarks get pods -o wide
+        # We only test the typical case case4, otherwise, it will take very long time.
+        # cases="case1 case2 case3 case4"
+        cases="case4"
```
Comment (h00130372) [inline playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml]: Or maybe we should use a variable to set this instead hard coding?

Comment (liu-sheng) [reply playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml]: done

File: playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml [hunk 2]
```diff
@@ -133,76 +136,73 @@
...
-        kubectl -n kf-tfbenchmarks get tfjobs -o wide
       args:
         executable: /bin/bash
         chdir: '{{ ansible_user_dir }}/workspace'
-      with_items:
-        # We only test the typical case case4, otherwise, it will take very long time.
-        # - case1
-        # - case2
-        # - case3
-        - case4
```
Comment (h00130372) [inline playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml]: Ditto.

File: playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/volcano_tfbenchmarks.yaml [hunk 3]
```diff
@@ -106,83 +109,80 @@
...
-        kubectl -n vc-tfbenchmarks get job.batch.volcano.sh
       args:
         executable: /bin/bash
         chdir: '{{ ansible_user_dir }}/workspace'
-      with_items:
-        # We only test the typical case case4, otherwise, it will take very long time.
-        # - case1
-        # - case2
-        # - case3
-        - case4
```
Comment (h00130372) [inline playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/volcano_tfbenchmarks.yaml]: Ditto.

File: playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml (hunk 1 above)
Comment (h00130372) [reply playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml]: Thanks. And it's better to change the two other places, see comments.

Comment (liu-sheng) [reply playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml]: also changed

**Pull request merged**